provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
```

Comment:
U `sandboxes-service` jsem odhalil novy, IMHO lepsi pristup jak s tim pracovat. Nereferencovat konkretni profil (protoze teoreticky ho muzeme mit pojmenovany ruzne), ale rict konkretne, ke kterymu AWS accountu potrebuju pristup (https://github.com/keboola/sandboxes-service/blob/8406d9dc004904dd374f2c6f3337f93945fe3358/provisioning/local/aws.tf#L2).

A jak jsem psal u README, nemusi to pak byt konfigurovatelne.